app/api.py

```diff
@@ -17,11 +18,34 @@ def fetch_from_api():
     """)
     url = 'https://parseapi.back4app.com/classes/Carmodels_Car_Model_List?limit=10&keys=Make,Year&where=%s' % where
     headers = {

```

Comment:
Can you try using python f-strings? they are more structured and easy to use.

Reply:
sure, boss.